### ISSUE TITLE
Date field gets rounded

### ISSUE THREAD

**Issue by aerni** [statamic/cms#3923]
## Bug Description
I've got a `date` field that gets populated from an external API. The date is saved as a timestamp. Whenever I save an entry the date will be rounded, which is very unfortunate.

```yaml
// Original value from the API
pledge_date: 1624875575
// 2021-06-28 10:19:35.0 +00:00

// Rounded value after save
pledge_date: 1624875540 
// 2021-06-28 10:19:00.0 +00:00
```

## How to Reproduce
1. Add a date field with `format: U` to your blueprint
2. Create an entry and add this value to the field: `1624875575`
3. Save the entry in the CP
4. The date will now be rounded to `1624875540`

## Environment
Statamic 3.1.26 Pro
Laravel 8.48.2
PHP 7.4.20
aerni/social-links 2.2.0
jonassiewertsen/statamic-livewire 2.5.0
visuellverstehen/statamic-classify 2.1.1

**Comment by arthurperton** [comment]
Closing, this has been solved in PR #3959.

**Comment by jasonvarga** [comment]
I'm revisiting some date fieldtype stuff and came across this issue.

I'm thinking that the behavior was actually correct. If you had not set `time_enabled: true` on the field, when you saved, the date would remain and the time would be wiped. Seems expected.
